packages/block-library/src/terms-query/inspector-controls/max-terms-controls.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	__experimentalToolsPanelItem as ToolsPanelItem,
+	RangeControl,
+} from '@wordpress/components';
+
+export default function MaxTermsControls( { attributes, setQuery } ) {
```

Comment:
I see this is being imported as `MaxTermsControl` (rather than `MaxTermsControls`) in `inspector-controls/index.js`, so we should make these names consistent. It looks like the other similar control components are singular, so I think I'd suggest making this one singular as well:

```suggestion
export default function MaxTermsControl( { attributes, setQuery } ) {
```

And then I'd also make the filename for this singular, too. Sorry, this feels incredibly picky, but everything else is so neat 😄

Reply:
Hahah, not at all! Be picky, let's keep it consistent! 

Done in https://github.com/WordPress/gutenberg/pull/71633/commits/af6df671e518a2c9de736f8c43fc9101dd89c232